integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb

```diff
@@ -1055,6 +1055,11 @@
    "provenance": [],
```

Comment:
Why this goes together? Shouldn't it go as a separate PR?

Reply:
I don't know what happened exactly, but this should not be here indeed.